historical-contributors-agreement.txt

```diff
@@ -234,6 +234,7 @@ YYYY/MM/DD, github id, Full name, email
 2019/08/27, wurzelpeter, Markus Franke, markus[hyphen]franke[at]web[dot]de
 2019/09/10, ImanHosseini, Iman Hosseini, hosseini.iman@yahoo.com
 2019/09/03, João Henrique, johnnyonflame@hotmail.com
+<<<<<<< HEAD
```

Comment:
This line looks like leftover of conflict resolving. Please remove.

Reply:
Oops, removed. Removed the last line too.